Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.10.7
+
+ENV PYTHONNUNBUFFERED 1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+
+CMD ["python", "manage.py", "runserver", "0.0.0.0:8000"]
```

Comment:
why runserver????

Reply:
why not?

Reply:
you `runserver` like this only in Django projects

Reply:
and this is not a django project it will not work

Reply:
forgot about it

Reply:
attach a screenshot of the task result from the terminal so that it can be seen that it is running in docker

Reply:
![image](https://github.com/mate-academy/py-docker-weather-api/assets/95485025/f158c3e6-d0a3-477c-8411-2dc82948685f)